manual/mapper/daos/select/README.md

```diff
@@ -60,6 +60,20 @@ for each, with the same name and a compatible Java type.
 PagingIterable<Product> findByDescription(String searchString);
 ```
 
+The generated SELECT query can be further customized with [limit()], [perPartitionLimit()],
+[orderBy()], [groupBy()] and [allowFiltering()]. Some of these clauses can also contain placeholders
+whose values will be provided through additional method parameters. Note that it is sometimes not
+possible to determine if a parameter is a primary key component or a placeholder value; therefore
+the rule is that **if your method takes a partial primary key, all other parameters must be
+explicitly annotated with [@CqlName](../../entities/#user-provided-names)**. For example if the
+primary key is `((day int, hour int, minute int), ts timestamp)`:
+
+```java
+// Annotate 'l' so that it's not mistaken for the second PK component
+@Select(limit = ":l")
+PagingIterable<Sale> findDailySales(int day, @CqlName("l") int l);
```

Comment:
I agree with the principle and am OK with it. Out of curiosity, have you considered the below alternative?

```
@Select
PagingIterable<Sale> findDailySales(int day, @Limit int limit);
```

Reply:
Yes, but we would need one such annotation for every secondary clause (`@PartitionLimit`, etc.), and this also assumes that there can only be one bind marker per clause. Currently this is the case, but this could change in the future (e.g. when `GROUP BY` accepts functions).